index.js

```diff
@@ -109,11 +109,8 @@ module.exports = function (app, options) {
     }
 
     // __(key, value1, value2, value3, value4, value5, ...)
-    var args = new Array(arguments.length);
+    var args = Array.prototype.slice.call(arguments);
     args[0] = text;
-    for(var i = 1; i < args.length; i++) {
-      args[i] = arguments[i];
-    }
```

Comment:
for loop will more faster than slice...

![image](https://cloud.githubusercontent.com/assets/156269/8782695/c04ad704-2f49-11e5-9295-c2d2cbea691c.png)